libraries/rush-lib/src/pluginFramework/PluginManager.ts

```diff
@@ -81,6 +81,7 @@ export class PluginManager {
     tryAddBuiltInPlugin('rush-amazon-s3-build-cache-plugin');
     tryAddBuiltInPlugin('rush-azure-storage-build-cache-plugin');
     tryAddBuiltInPlugin('rush-http-build-cache-plugin');
+    tryAddBuiltInPlugin('rush-bridge-cache-plugin');
```

Comment:
This plugin shouldn't be installed by default, so make sure this gets removed before checkin.